spring-cloud-netflix-core/src/test/resources/application.yml

```diff
@@ -30,6 +30,9 @@ badClients:
   ribbon:
     MaxAutoRetriesNextServer: 10
     ReadTimeout: 200
+username:
+  ribbon:
+    listOfServers: localhost:${local.server.port}
```

Comment:
Does this work? If so, it surprises me because I didn't think it was available in time.